widget/view.controller.js

```diff
@@ -9,39 +9,88 @@
+  function threatIntelManagementConfiguration100Ctrl($scope, threatIntelManagementConfigurationService, widgetDataIngestionService, WizardHandler, $controller, $state, connectorService, currentPermissionsService, CommonUtils, API, _, $filter, $http, dataIngestionService, PagedCollection, $resource, FIXED_MODULE, Entity, playbookService, translationService, toaster, appModulesService, widgetBasePath, $rootScope, $timeout, ALL_RECORDS_SIZE) {
     $controller('BaseConnectorCtrl', {
       $scope: $scope
     });
     $scope.backToStartPage = backToStartPage;
+    $scope.backToConfigConnector = backToConfigConnector;
     $scope.moveToConfigureConnector = moveToConfigureConnector;
     $scope.backToinstallConnector = backToinstallConnector;
     $scope.moveToSelectConnector = moveToSelectConnector;
+    $scope.backToSelectConnector = backToSelectConnector;
+    $scope.moveToFeedRules = moveToFeedRules;
+    $scope.backToFeedRules = backToFeedRules;
+    $scope.configFieldChanged = configFieldChanged;
+    $scope.moveToFinish = moveToFinish;
     $scope.installConnector = installConnector;
+    $scope.close = close;
     $scope.toggleSelectFeedsSettings = toggleSelectFeedsSettings;
     $scope.toggleConnectorConfigSettings = toggleConnectorConfigSettings;
+    $scope.dataIngestionParamsUpdating = false;
+    $scope.areFeedConnectorsConfigured = false;
+    $scope.toggleConnectorConfig = [];
+    $scope.toggleParametersConfig = [];
+    $scope.toggleScheduleConfig = [];
+    $scope.toggleParametersSettings = toggleParametersSettings;
+    $scope.toggleScheduleConfigSettings = toggleScheduleConfigSettings;
+    $scope.toggleFeedRules = toggleFeedRules;
+    $scope.saveParams = saveParams;
     $scope.allConnectorsInstalled = false;
+    $scope.loadActiveTab = loadActiveTab;
     $scope.feedConnectors = [];
+    $scope.healthyConnectors = [];
     $scope.installedConnectors = [];
+    $scope.samplePlaybookEntity = {};
+    $scope.ingestMethodActions = {};
     $scope.searchQuery = '';
+    $scope.healthyConnectorsParams = [];
+    $scope.connectorHealthStatus = [];
+    $scope.connectorParamsStatus = [];
+    $scope.scheduleJsonData = undefined;
+    $scope.params = {
+      activeTab: 0
+    };
+    $scope.feedRules = {
+      highConfidenceThreatFeeds: {
+        enabled: true,
+        open: true,
+        name: 'High Confidence Threat Feed Block',
+        feedConfidenceThreshold: 70
+      },
+      feedToIndicatorLinking: {
+        enabled: true,
+        open: true,
+        name: 'Feed To Indicator Linking',
+        feedConfidenceThreshold: 70
+      },
+      unstructuredFeedsSupport: {
+        enabled: true,
+        open: true,
+        name: 'Unstructured Feeds Support',
+        ingestFeedsFromFile: true,
+        ingestFeedsFromEmail: false
+      }
+    };
+    $scope.connectorFetchIndex = {};
+    $scope.connectorConfigIndex = {};
```

Comment:
remove unused variables connectorFetchIndex and connectorConfigIndex

Reply:
As discussed passed through scope

Reply:
used in data ingestion component